ui/app/components/transit-key-actions.js

```diff
@@ -130,6 +130,15 @@ export default Component.extend(TRANSIT_PARAMS, {
     this.set('errors', e.errors);
   },
 
+  clearErrors() {
+    this.set('errors', null);
+  },
+
+  /* eslint-disable-next-line ember/no-observers */
+  selectedActionChanged: observer('selectedAction', function() {
```

Comment:
what about setting `errors` to `null` inside `checkAction` or `resetParams`? seems like that might be a good alternative to using an observer since it appears those functions are called when the user clicks on a new action. :D

Reply:
Brilliant, love it! I'll test that and make sure the tests are a go :D